data_model/src/domain.rs

```diff
@@ -169,8 +188,28 @@ pub struct Domain {
     metadata: Metadata,
```

Comment:
I'd very much like to interact with the bindgen a little. Since i'll mostly be working on this over the weekend, I'll have a bunch of questions by Monday.

Reply:
Yeah, I'm just curious why it doesn't cause conflicts in the FFI.

Reply:
Done